src/bitwarden_exporter/settings.py

```
@@ -103,17 +104,38 @@ def get_bitwarden_settings_based_on_args() -> BitwardenExportSettings:
-        "This will not delete the temporary directory after the export,"
-        " Default: --no-debug",
+        help=cli_debug_help,
         action=argparse.BooleanOptionalAction,
         default=False,
     )
 
-    if __name__ == "__main__":
-        print(pyfiglet.figlet_format("Bitwarden Exporter"))
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), "resources", "version.txt"),
+        "r",
```

Ruff (0.14.1)

Unnecessary mode argument

Remove mode argument

(UP015)